docs/source/apis/api_leveling.md

```diff
@@ -0,0 +1,72 @@
+# Llama Stack API Stability Leveling
+
+In order to provide a stable experience in Llama Stack, the various APIs need different stability levels indicating the level of support, backwards compatability, and overall production readiness.
+
+## Different Levels
+
+### v1alpha
+
+- Little to no expectation of support between versions
+- Breaking changes are permitted
+- Datatypes and parameters can break
+- Routes can be added and removed
+
+#### Graduation Criteria
+
+- an API can graduate from `v1alpha` to `v1beta` if the API surface is complete. Meaning no net-new routes will be added that all providers must implement.
```

Comment:
```suggestion
- an API can graduate from `v1alpha` to `v1beta` if the team has identified the minimum set of capabilities every provider must implement. Also CRUD must stay stable once in beta.
```

I think the "no net-new" is a bit unrealistic and more aspirational than enforceable; most APIs don’t stop growing once they hit beta or GA.  But optional new routes/extensions are fine.

Reply:
yeah this is fair, will change